server/models/list.js

```diff
@@ -0,0 +1,57 @@
+const _ = require('builders/utils')
+const assert_ = require('lib/utils/assert_types')
+const validations = require('./validations/list')
+const attributes = require('./attributes/list')
+const error_ = require('lib/error/error')
+
+module.exports = {
+  create: list => {
```

Comment:
those model functions could have some unit tests

Reply:
i dont really get the value of thos model tests for such a straight forward endpoint

but here you go 7c31b287